File: Travelo.Model/Trip.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Travelo.Model
{
    public partial class Trip
    {
        public int Id { get; set; }
        public byte[] AccomodationImage { get; set; }
        public string AccomodationName { get; set; }
        public string AccomodationDescription { get; set; }
        public int Rating { get; set; } = 5;
        public int RatingCount { get; set; } = 115;
        public List<string> Facilities { get; set; }
        public List<string> AllDates { get; set; }
        public string Location { get; set; }
        public string Dates { get; set; }
        public int LowestPrice { get; set; }
        public int HighestPrice { get; set; }
        public int AgencyId { get; set; }
        public string AgencyName { get; set; }
        public byte[] AgencyImage { get; set; }
        public string CountryName { get; set; }
        public string CityName { get; set; }
        public bool Bookamrked { get; set; }
        public int AccomodationId { get; set; }

    } 

}

File: travelo_agency/lib/models/trip.dart
import 'package:json_annotation/json_annotation.dart';

part 'trip.g.dart';

@JsonSerializable()
class Trip {
  int? id;
  String? accomodationImage;
  String? accomodationName;
  String? accomodationDescription;
  int? accomodationId;
  int? rating;
  int? ratingCount;
  List<String>? facilities;
  List<String>? allDates;
  String? location;
  String? dates;
  int? lowestPrice;
  int? highestPrice;
  int? agencyId;
  String? agencyName;
  String? agencyImage;
  String? countryName;
  String? cityName;
  bool? bookamrked;

  Trip();

  factory Trip.fromJson(Map<String, dynamic> json) => _$TripFromJson(json);

  // Map toJson() => {
  //       'id': id,
  //       'name': name,
  //       'countryName': countryName,
  //       'tags': tags,
  //       'image': image,
  //       'numberOfTrips': numberOfTrips,
  //       'lowestTripPrice': lowestTripPrice,
  //     };
}
